src/clib/xtg/grd3d_roff2xtgeo_splitenz.c

```diff
@@ -0,0 +1,98 @@
+/*
+ ***************************************************************************************
+ *
+ * NAME:
+ *    grd3d_roff2xtgeo_zcorn.c
+ *
+ * DESCRIPTION:
+ *    Convert from ROFF arrays to XTGeo arrays: ZCORN
+ *
+ * ARGUMENTS:
+ *    nx, ny, nz       i     NLAY dimens
```

Comment:
"NLAY dimens" replace with "Dimension (nx=ncol, ny=nrow, nz=nlay)"

(I generally find the wording NCOL better than NX, but keep as is)

Reply:
Fixed!